src/v/kafka/server/connection_context.h

```diff
@@ -82,6 +85,7 @@ class connection_context final
     sequence_id _seq_idx;
     map_t _responses;
     sasl_server _sasl;
+    const ss::sstring _client_addr;
```

Comment:
interesting. should we actually just store the ip as a uint32 instead of doing character per character matching? 

should this be actually a `ss::ip_addr` struct that can do ipv6 and ip4 and have all the smarts to compare with integers?

Reply:
I would suggest the same, having this address will be helpful to build readers cache for fetches.

Reply:
fixed